client/src/components/SignupForm.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import {
+  Form, Input, Button, message,
+} from 'antd';
+import { Link } from 'react-router-dom';
+import httpService from '../services/httpService';
+import Img from './common/Img';
+import ParagraphText from './common/Paragraph';
+import HeaderTitle from './common/Title';
+import 'antd/dist/antd.css';
+import './signup.css';
+
+function SignupForm() {
+  const [errorMsg, setErrorMsg] = useState('');
+  const errorlert = (erroMsg) => {
+    message.error(erroMsg);
+  };
+  const postData = async (userInfo) => {
+    try {
+      const { msg } = await httpService.post('/api/v1/signup', userInfo);
+      setErrorMsg(msg);
+    } catch (error) {
+      setErrorMsg(error);
+    }
+  };
+
+  const onFinish = (values) => {
+    const { confirmPassword, password } = values;
+    if (confirmPassword === password) {
+      postData(values);
+    }
+  };
+  return (
+    <div className="signupContainer">
+
+      <div className="signup">
+        {errorMsg ? errorlert(errorMsg) : ''}
+        <Form
+          name="basic"
+          labelCol={{ span: 8 }}
+          wrapperCol={{ span: 7 }}
+          initialValues={{ remember: true }}
+          autoComplete="off"
+          onFinish={onFinish}
+          className="signupForm"
+        >
+          <HeaderTitle text="إنشاء حساب" level={2} className="signupTitle" />
+          <Form.Item
+            name="name"
+            rules={[{ required: true, message: 'اسم المستخدم مطلوب' }]}
+          >
+            <Input placeholder="اسم المستخدم " className="signupInput" />
+          </Form.Item>
+          <Form.Item
+            name="email"
+            rules={[{ required: true, message: 'البريد الإلكتروني مطلوب' }]}
+          >
+            <Input placeholder=" البريد الإلكتروني  " className="signupInput" />
+          </Form.Item>
+
+          <Form.Item
+            placeholder="كلمة المرور "
+            name="password"
+            rules={[{ required: true, message: 'كلمة المرور مطلوبة' }]}
+          >
+            <Input.Password placeholder=" كلمة المرور " className="signupInput" />
+          </Form.Item>
+          <Form.Item
+            name="confirmPassword"
+            rules={[{ required: true, message: '  تأكيد كلمة المرور مطلوبة' }]}
+          >
+            <Input.Password placeholder=" تأكيد كلمة المرور " className="signupInput" />
```

Comment:
here, the same thing, you can destruct password from input

Reply:
done